### PR TITLE
Add TVL Adapter: Equity

### PR DESCRIPTION
**Pull request by ftm1337**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Equity

##### Twitter Link:
https://twitter.com/Equalizer0x

##### List of audit links if any:
n.a.

##### Website Link:
https://equity.equalizer.exchange

##### Logo (High resolution, will be shown with rounded borders):
https://ftm.guru/icons/equity.png

##### Current TVL:
$1500

##### Treasury Addresses (if the protocol has treasury)
0xc424c343554afd6cd270887d4232765850f5e93f

##### Chain:
Fantom

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
equalizer-dex 

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
4088

##### Short Description (to be shown on DefiLlama):
Equity offers a complete suite of Decentralized Exchange products that rewards veEQUAL Governance NFT holders with Full 100% fees stemming from Five Sources:
-    Market-making (Liquidity Provisioning)
-    Spot Trading (Simple Exchange)
-    Flash Loans (Instant repayments)
-    Derivatives Trading (Perpetual Swaps)
-    Liquidations (Settlement Fees)

##### Token address and ticker if any:
- EQUITY
fantom:0x180DA8bD91406Dd41F18b6200C53dEb553f86180
##### Category (full list at https://defillama.com/categories) *Please choose only one:
DEX

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
TWAP+Chainlink

##### forkedFrom (Does your project originate from another project):
GMX (Trading) + Solidly (Governance)

##### methodology (what is being counted as tvl, how is tvl being calculated):
All Equity Trading Vaults holding assets are counted towards TVL.

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/equity-spot exports TVL: 
        
 
 
    fantom                    1.74 k
    
    total                    1.74 k

File: projects/equity-spot/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Equity Spot - Central Vaults
+
+const sdk = require("@defillama/sdk")
+const { sumTokens } = require('../helper/unwrapLPs')
+
+const fantomVault_01 = "0x9e4105f9e2284532474f69e65680e440f4c91cb8";
+const chain= 'fantom'
+
+const fantomTVL = async (timestamp, _block, { [chain]: block}) => {
```
Comment (g1nt0ki) [inline projects/equity-spot/index.js:9]: hi @ftm1337 you can use our helper for gmx tvl: 

    const { gmxExports } = require("../helper/gmx");

    module.exports = {
      fantom: {
        tvl: gmxExports({ vault: '0x9e4105f9e2284532474f69e65680e440f4c91cb8' }),
      },
    };

Comment (ftm1337) [reply projects/equity-spot/index.js:9]: oh thats nice! thanks fren!

**Comment by llamatester** [comment]
The adapter at projects/equity-spot exports TVL: 
        
 
 
    fantom                    1.75 k
    
    total                    1.75 k

**Comment by llamatester** [comment]
The adapter at projects/equity exports TVL: 
        
 
 
    fantom                    2.06 k
    
    total                    2.06 k

**Comment by g1nt0ki** [comment]
is this ready for review/merge?

**Comment by ftm1337** [comment]
its almost ready, just waiting for the "official announcement".. dont want the defillama tweet to frontrun it :sweat_smile: 
closing this for now

**Comment by ftm1337** [comment]
Good to merge :handshake:

**Comment by llamatester** [comment]
The adapter at projects/equity exports TVL: 
        
 
 
    fantom                    2.06 k
    
    total                    2.06 k

**Comment by Define101** [comment]
When will the project launch? Website is not working

**Comment by ftm1337** [comment]
@Define101 it soft launched yesterday, had some dns issues on the expected subdomain, and was hosted at a temp url.
its live on the main one since earlier today : )

**Pull request merged**